src/content/docs/tutorial-diagnose-infra-bottlenecks/instrument-your-system.mdx

```diff
@@ -0,0 +1,125 @@
+---
+title: Collect data about your system infrastructure
+metaDescription: Capture data about your system by installing agents, then view that data in New Relic. 
+---
+
+Like the bridges and roads of a dense city, your system infrastructure needs regular inspection and maintenance. What processes affect CPU and disk usage? What app behaviors typically correlate with failing hosts? The performance of your infrastructure can affect the performance of your APIs, services, and apps. A database failure can prevent your server from handling data, or a memory leak can bring an otherwise performant app to an abrupt halt. 
+
+## Objectives [#objectives]
+
+This tutorial series walks you through using infrastructure data to troubleshoot different incident scenarios. While New Relic offers a full suite integrations to monitor the Cloud, containerized environments, and various virtual machines, this series is specific to data reported with an infrastructure agent that does not have additional integrations.
+
+That said, this doc explains how you might fully instrument your system. You will:
+
+* Install the infrastructure agent with our guided install
+* Understand the guided install's recommendations so you can make an informed decision about instrumentation
+* Learn about infrastructure integrations 
+
+## Install the infrastructure agent  [#install]
+
+When you observe your infrastructure in context, you can respond to incidents with more clarity and direction, letting you move from reaction to proaction in your incident response.
```

Comment:
Is it worth explaining what infrastructure _is_. I think there are opportunities in the intro to go into what makes infrastructure different than just monitoring w/ APM.